obspy/io/nordic/core.py

```diff
@@ -798,6 +814,35 @@ def _read_comments(tagged_lines, event):
     return event
 
 
+def _read_event_id(tagged_lines, event):
+    """
+    Internal reader for ID line.
+
+    :type tagged_lines: dict
+    :param tagged_lines: dictionary of tagged lines
+    :type event: :class:`~obspy.core.event.event.Event`
+    :param event: Event to associate spectral info with
+
+    :returns:
+        event with event.extra.nordic_event_id set to event ID from Nordic file
+    """
+    if 'I' not in tagged_lines.keys():
+        return event
+    id_lines = tagged_lines['I']
+    if len(id_lines) > 1:
+        warnings.warn('Nordic file has more than one ID line, will use first' +
+                      'ID line only.')
+    for id_line, tag in id_lines:
+        event_id = id_line.split('ID:')[-1].split(' ')[0].strip('dSLRD')
+        break
+    event.extra = {
```

Comment:
Right now this might be the only instance setting anything on `extra` but that might change.. might be safer to do something like 

```python
extra = event.setdefault('extra', {})
extra['nordic_event_id'] = ...
```